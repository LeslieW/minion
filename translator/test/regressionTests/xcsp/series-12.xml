<?xml version="1.0" encoding="UTF-8"?>

<instance>
<presentation name="?" maxConstraintArity="3" format="XCSP 2.0"/>

<domains nbDomains="2">
<domain name="D0" nbValues="12">0..11</domain>
<domain name="D1" nbValues="11">0..10</domain>
</domains>

<variables nbVariables="23">
<variable name="V0" domain="D0"/>
<variable name="V1" domain="D0"/>
<variable name="V2" domain="D0"/>
<variable name="V3" domain="D0"/>
<variable name="V4" domain="D0"/>
<variable name="V5" domain="D0"/>
<variable name="V6" domain="D0"/>
<variable name="V7" domain="D0"/>
<variable name="V8" domain="D0"/>
<variable name="V9" domain="D0"/>
<variable name="V10" domain="D0"/>
<variable name="V11" domain="D0"/>
<variable name="V12" domain="D1"/>
<variable name="V13" domain="D1"/>
<variable name="V14" domain="D1"/>
<variable name="V15" domain="D1"/>
<variable name="V16" domain="D1"/>
<variable name="V17" domain="D1"/>
<variable name="V18" domain="D1"/>
<variable name="V19" domain="D1"/>
<variable name="V20" domain="D1"/>
<variable name="V21" domain="D1"/>
<variable name="V22" domain="D1"/>
</variables>

<predicates nbPredicates="2">
<predicate name="P0">
  <parameters>int X0 int X1</parameters>
  <expression> 
    <functional>ne(X0,X1)</functional>
  </expression>
</predicate>
<predicate name="P1">
  <parameters>int X0 int X1 int X2 int X3</parameters>
  <expression> 
    <functional>eq(add(X0,X1),abs(sub(X2,X3)))</functional>
  </expression>
</predicate>
</predicates>

<constraints nbConstraints="132">
<constraint name="C0" arity="2" scope="V0 V1" reference="P0">
  <parameters>V0 V1</parameters>
</constraint>
<constraint name="C1" arity="2" scope="V0 V2" reference="P0">
  <parameters>V0 V2</parameters>
</constraint>
<constraint name="C2" arity="2" scope="V0 V3" reference="P0">
  <parameters>V0 V3</parameters>
</constraint>
<constraint name="C3" arity="2" scope="V0 V4" reference="P0">
  <parameters>V0 V4</parameters>
</constraint>
<constraint name="C4" arity="2" scope="V0 V5" reference="P0">
  <parameters>V0 V5</parameters>
</constraint>
<constraint name="C5" arity="2" scope="V0 V6" reference="P0">
  <parameters>V0 V6</parameters>
</constraint>
<constraint name="C6" arity="2" scope="V0 V7" reference="P0">
  <parameters>V0 V7</parameters>
</constraint>
<constraint name="C7" arity="2" scope="V0 V8" reference="P0">
  <parameters>V0 V8</parameters>
</constraint>
<constraint name="C8" arity="2" scope="V0 V9" reference="P0">
  <parameters>V0 V9</parameters>
</constraint>
<constraint name="C9" arity="2" scope="V0 V10" reference="P0">
  <parameters>V0 V10</parameters>
</constraint>
<constraint name="C10" arity="2" scope="V0 V11" reference="P0">
  <parameters>V0 V11</parameters>
</constraint>
<constraint name="C11" arity="2" scope="V1 V2" reference="P0">
  <parameters>V1 V2</parameters>
</constraint>
<constraint name="C12" arity="2" scope="V1 V3" reference="P0">
  <parameters>V1 V3</parameters>
</constraint>
<constraint name="C13" arity="2" scope="V1 V4" reference="P0">
  <parameters>V1 V4</parameters>
</constraint>
<constraint name="C14" arity="2" scope="V1 V5" reference="P0">
  <parameters>V1 V5</parameters>
</constraint>
<constraint name="C15" arity="2" scope="V1 V6" reference="P0">
  <parameters>V1 V6</parameters>
</constraint>
<constraint name="C16" arity="2" scope="V1 V7" reference="P0">
  <parameters>V1 V7</parameters>
</constraint>
<constraint name="C17" arity="2" scope="V1 V8" reference="P0">
  <parameters>V1 V8</parameters>
</constraint>
<constraint name="C18" arity="2" scope="V1 V9" reference="P0">
  <parameters>V1 V9</parameters>
</constraint>
<constraint name="C19" arity="2" scope="V1 V10" reference="P0">
  <parameters>V1 V10</parameters>
</constraint>
<constraint name="C20" arity="2" scope="V1 V11" reference="P0">
  <parameters>V1 V11</parameters>
</constraint>
<constraint name="C21" arity="2" scope="V2 V3" reference="P0">
  <parameters>V2 V3</parameters>
</constraint>
<constraint name="C22" arity="2" scope="V2 V4" reference="P0">
  <parameters>V2 V4</parameters>
</constraint>
<constraint name="C23" arity="2" scope="V2 V5" reference="P0">
  <parameters>V2 V5</parameters>
</constraint>
<constraint name="C24" arity="2" scope="V2 V6" reference="P0">
  <parameters>V2 V6</parameters>
</constraint>
<constraint name="C25" arity="2" scope="V2 V7" reference="P0">
  <parameters>V2 V7</parameters>
</constraint>
<constraint name="C26" arity="2" scope="V2 V8" reference="P0">
  <parameters>V2 V8</parameters>
</constraint>
<constraint name="C27" arity="2" scope="V2 V9" reference="P0">
  <parameters>V2 V9</parameters>
</constraint>
<constraint name="C28" arity="2" scope="V2 V10" reference="P0">
  <parameters>V2 V10</parameters>
</constraint>
<constraint name="C29" arity="2" scope="V2 V11" reference="P0">
  <parameters>V2 V11</parameters>
</constraint>
<constraint name="C30" arity="2" scope="V3 V4" reference="P0">
  <parameters>V3 V4</parameters>
</constraint>
<constraint name="C31" arity="2" scope="V3 V5" reference="P0">
  <parameters>V3 V5</parameters>
</constraint>
<constraint name="C32" arity="2" scope="V3 V6" reference="P0">
  <parameters>V3 V6</parameters>
</constraint>
<constraint name="C33" arity="2" scope="V3 V7" reference="P0">
  <parameters>V3 V7</parameters>
</constraint>
<constraint name="C34" arity="2" scope="V3 V8" reference="P0">
  <parameters>V3 V8</parameters>
</constraint>
<constraint name="C35" arity="2" scope="V3 V9" reference="P0">
  <parameters>V3 V9</parameters>
</constraint>
<constraint name="C36" arity="2" scope="V3 V10" reference="P0">
  <parameters>V3 V10</parameters>
</constraint>
<constraint name="C37" arity="2" scope="V3 V11" reference="P0">
  <parameters>V3 V11</parameters>
</constraint>
<constraint name="C38" arity="2" scope="V4 V5" reference="P0">
  <parameters>V4 V5</parameters>
</constraint>
<constraint name="C39" arity="2" scope="V4 V6" reference="P0">
  <parameters>V4 V6</parameters>
</constraint>
<constraint name="C40" arity="2" scope="V4 V7" reference="P0">
  <parameters>V4 V7</parameters>
</constraint>
<constraint name="C41" arity="2" scope="V4 V8" reference="P0">
  <parameters>V4 V8</parameters>
</constraint>
<constraint name="C42" arity="2" scope="V4 V9" reference="P0">
  <parameters>V4 V9</parameters>
</constraint>
<constraint name="C43" arity="2" scope="V4 V10" reference="P0">
  <parameters>V4 V10</parameters>
</constraint>
<constraint name="C44" arity="2" scope="V4 V11" reference="P0">
  <parameters>V4 V11</parameters>
</constraint>
<constraint name="C45" arity="2" scope="V5 V6" reference="P0">
  <parameters>V5 V6</parameters>
</constraint>
<constraint name="C46" arity="2" scope="V5 V7" reference="P0">
  <parameters>V5 V7</parameters>
</constraint>
<constraint name="C47" arity="2" scope="V5 V8" reference="P0">
  <parameters>V5 V8</parameters>
</constraint>
<constraint name="C48" arity="2" scope="V5 V9" reference="P0">
  <parameters>V5 V9</parameters>
</constraint>
<constraint name="C49" arity="2" scope="V5 V10" reference="P0">
  <parameters>V5 V10</parameters>
</constraint>
<constraint name="C50" arity="2" scope="V5 V11" reference="P0">
  <parameters>V5 V11</parameters>
</constraint>
<constraint name="C51" arity="2" scope="V6 V7" reference="P0">
  <parameters>V6 V7</parameters>
</constraint>
<constraint name="C52" arity="2" scope="V6 V8" reference="P0">
  <parameters>V6 V8</parameters>
</constraint>
<constraint name="C53" arity="2" scope="V6 V9" reference="P0">
  <parameters>V6 V9</parameters>
</constraint>
<constraint name="C54" arity="2" scope="V6 V10" reference="P0">
  <parameters>V6 V10</parameters>
</constraint>
<constraint name="C55" arity="2" scope="V6 V11" reference="P0">
  <parameters>V6 V11</parameters>
</constraint>
<constraint name="C56" arity="2" scope="V7 V8" reference="P0">
  <parameters>V7 V8</parameters>
</constraint>
<constraint name="C57" arity="2" scope="V7 V9" reference="P0">
  <parameters>V7 V9</parameters>
</constraint>
<constraint name="C58" arity="2" scope="V7 V10" reference="P0">
  <parameters>V7 V10</parameters>
</constraint>
<constraint name="C59" arity="2" scope="V7 V11" reference="P0">
  <parameters>V7 V11</parameters>
</constraint>
<constraint name="C60" arity="2" scope="V8 V9" reference="P0">
  <parameters>V8 V9</parameters>
</constraint>
<constraint name="C61" arity="2" scope="V8 V10" reference="P0">
  <parameters>V8 V10</parameters>
</constraint>
<constraint name="C62" arity="2" scope="V8 V11" reference="P0">
  <parameters>V8 V11</parameters>
</constraint>
<constraint name="C63" arity="2" scope="V9 V10" reference="P0">
  <parameters>V9 V10</parameters>
</constraint>
<constraint name="C64" arity="2" scope="V9 V11" reference="P0">
  <parameters>V9 V11</parameters>
</constraint>
<constraint name="C65" arity="2" scope="V10 V11" reference="P0">
  <parameters>V10 V11</parameters>
</constraint>
<constraint name="C66" arity="2" scope="V12 V13" reference="P0">
  <parameters>V12 V13</parameters>
</constraint>
<constraint name="C67" arity="2" scope="V12 V14" reference="P0">
  <parameters>V12 V14</parameters>
</constraint>
<constraint name="C68" arity="2" scope="V12 V15" reference="P0">
  <parameters>V12 V15</parameters>
</constraint>
<constraint name="C69" arity="2" scope="V12 V16" reference="P0">
  <parameters>V12 V16</parameters>
</constraint>
<constraint name="C70" arity="2" scope="V12 V17" reference="P0">
  <parameters>V12 V17</parameters>
</constraint>
<constraint name="C71" arity="2" scope="V12 V18" reference="P0">
  <parameters>V12 V18</parameters>
</constraint>
<constraint name="C72" arity="2" scope="V12 V19" reference="P0">
  <parameters>V12 V19</parameters>
</constraint>
<constraint name="C73" arity="2" scope="V12 V20" reference="P0">
  <parameters>V12 V20</parameters>
</constraint>
<constraint name="C74" arity="2" scope="V12 V21" reference="P0">
  <parameters>V12 V21</parameters>
</constraint>
<constraint name="C75" arity="2" scope="V12 V22" reference="P0">
  <parameters>V12 V22</parameters>
</constraint>
<constraint name="C76" arity="2" scope="V13 V14" reference="P0">
  <parameters>V13 V14</parameters>
</constraint>
<constraint name="C77" arity="2" scope="V13 V15" reference="P0">
  <parameters>V13 V15</parameters>
</constraint>
<constraint name="C78" arity="2" scope="V13 V16" reference="P0">
  <parameters>V13 V16</parameters>
</constraint>
<constraint name="C79" arity="2" scope="V13 V17" reference="P0">
  <parameters>V13 V17</parameters>
</constraint>
<constraint name="C80" arity="2" scope="V13 V18" reference="P0">
  <parameters>V13 V18</parameters>
</constraint>
<constraint name="C81" arity="2" scope="V13 V19" reference="P0">
  <parameters>V13 V19</parameters>
</constraint>
<constraint name="C82" arity="2" scope="V13 V20" reference="P0">
  <parameters>V13 V20</parameters>
</constraint>
<constraint name="C83" arity="2" scope="V13 V21" reference="P0">
  <parameters>V13 V21</parameters>
</constraint>
<constraint name="C84" arity="2" scope="V13 V22" reference="P0">
  <parameters>V13 V22</parameters>
</constraint>
<constraint name="C85" arity="2" scope="V14 V15" reference="P0">
  <parameters>V14 V15</parameters>
</constraint>
<constraint name="C86" arity="2" scope="V14 V16" reference="P0">
  <parameters>V14 V16</parameters>
</constraint>
<constraint name="C87" arity="2" scope="V14 V17" reference="P0">
  <parameters>V14 V17</parameters>
</constraint>
<constraint name="C88" arity="2" scope="V14 V18" reference="P0">
  <parameters>V14 V18</parameters>
</constraint>
<constraint name="C89" arity="2" scope="V14 V19" reference="P0">
  <parameters>V14 V19</parameters>
</constraint>
<constraint name="C90" arity="2" scope="V14 V20" reference="P0">
  <parameters>V14 V20</parameters>
</constraint>
<constraint name="C91" arity="2" scope="V14 V21" reference="P0">
  <parameters>V14 V21</parameters>
</constraint>
<constraint name="C92" arity="2" scope="V14 V22" reference="P0">
  <parameters>V14 V22</parameters>
</constraint>
<constraint name="C93" arity="2" scope="V15 V16" reference="P0">
  <parameters>V15 V16</parameters>
</constraint>
<constraint name="C94" arity="2" scope="V15 V17" reference="P0">
  <parameters>V15 V17</parameters>
</constraint>
<constraint name="C95" arity="2" scope="V15 V18" reference="P0">
  <parameters>V15 V18</parameters>
</constraint>
<constraint name="C96" arity="2" scope="V15 V19" reference="P0">
  <parameters>V15 V19</parameters>
</constraint>
<constraint name="C97" arity="2" scope="V15 V20" reference="P0">
  <parameters>V15 V20</parameters>
</constraint>
<constraint name="C98" arity="2" scope="V15 V21" reference="P0">
  <parameters>V15 V21</parameters>
</constraint>
<constraint name="C99" arity="2" scope="V15 V22" reference="P0">
  <parameters>V15 V22</parameters>
</constraint>
<constraint name="C100" arity="2" scope="V16 V17" reference="P0">
  <parameters>V16 V17</parameters>
</constraint>
<constraint name="C101" arity="2" scope="V16 V18" reference="P0">
  <parameters>V16 V18</parameters>
</constraint>
<constraint name="C102" arity="2" scope="V16 V19" reference="P0">
  <parameters>V16 V19</parameters>
</constraint>
<constraint name="C103" arity="2" scope="V16 V20" reference="P0">
  <parameters>V16 V20</parameters>
</constraint>
<constraint name="C104" arity="2" scope="V16 V21" reference="P0">
  <parameters>V16 V21</parameters>
</constraint>
<constraint name="C105" arity="2" scope="V16 V22" reference="P0">
  <parameters>V16 V22</parameters>
</constraint>
<constraint name="C106" arity="2" scope="V17 V18" reference="P0">
  <parameters>V17 V18</parameters>
</constraint>
<constraint name="C107" arity="2" scope="V17 V19" reference="P0">
  <parameters>V17 V19</parameters>
</constraint>
<constraint name="C108" arity="2" scope="V17 V20" reference="P0">
  <parameters>V17 V20</parameters>
</constraint>
<constraint name="C109" arity="2" scope="V17 V21" reference="P0">
  <parameters>V17 V21</parameters>
</constraint>
<constraint name="C110" arity="2" scope="V17 V22" reference="P0">
  <parameters>V17 V22</parameters>
</constraint>
<constraint name="C111" arity="2" scope="V18 V19" reference="P0">
  <parameters>V18 V19</parameters>
</constraint>
<constraint name="C112" arity="2" scope="V18 V20" reference="P0">
  <parameters>V18 V20</parameters>
</constraint>
<constraint name="C113" arity="2" scope="V18 V21" reference="P0">
  <parameters>V18 V21</parameters>
</constraint>
<constraint name="C114" arity="2" scope="V18 V22" reference="P0">
  <parameters>V18 V22</parameters>
</constraint>
<constraint name="C115" arity="2" scope="V19 V20" reference="P0">
  <parameters>V19 V20</parameters>
</constraint>
<constraint name="C116" arity="2" scope="V19 V21" reference="P0">
  <parameters>V19 V21</parameters>
</constraint>
<constraint name="C117" arity="2" scope="V19 V22" reference="P0">
  <parameters>V19 V22</parameters>
</constraint>
<constraint name="C118" arity="2" scope="V20 V21" reference="P0">
  <parameters>V20 V21</parameters>
</constraint>
<constraint name="C119" arity="2" scope="V20 V22" reference="P0">
  <parameters>V20 V22</parameters>
</constraint>
<constraint name="C120" arity="2" scope="V21 V22" reference="P0">
  <parameters>V21 V22</parameters>
</constraint>
<constraint name="C121" arity="3" scope="V12 V0 V1" reference="P1">
  <parameters>V12 1 V0 V1</parameters>
</constraint>
<constraint name="C122" arity="3" scope="V13 V1 V2" reference="P1">
  <parameters>V13 1 V1 V2</parameters>
</constraint>
<constraint name="C123" arity="3" scope="V14 V2 V3" reference="P1">
  <parameters>V14 1 V2 V3</parameters>
</constraint>
<constraint name="C124" arity="3" scope="V15 V3 V4" reference="P1">
  <parameters>V15 1 V3 V4</parameters>
</constraint>
<constraint name="C125" arity="3" scope="V16 V4 V5" reference="P1">
  <parameters>V16 1 V4 V5</parameters>
</constraint>
<constraint name="C126" arity="3" scope="V17 V5 V6" reference="P1">
  <parameters>V17 1 V5 V6</parameters>
</constraint>
<constraint name="C127" arity="3" scope="V18 V6 V7" reference="P1">
  <parameters>V18 1 V6 V7</parameters>
</constraint>
<constraint name="C128" arity="3" scope="V19 V7 V8" reference="P1">
  <parameters>V19 1 V7 V8</parameters>
</constraint>
<constraint name="C129" arity="3" scope="V20 V8 V9" reference="P1">
  <parameters>V20 1 V8 V9</parameters>
</constraint>
<constraint name="C130" arity="3" scope="V21 V9 V10" reference="P1">
  <parameters>V21 1 V9 V10</parameters>
</constraint>
<constraint name="C131" arity="3" scope="V22 V10 V11" reference="P1">
  <parameters>V22 1 V10 V11</parameters>
</constraint>
</constraints>
</instance>
