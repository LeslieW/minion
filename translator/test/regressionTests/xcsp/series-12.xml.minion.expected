MINION 3

#This file has been automatically generated by TAILOR v0.2
#bug-reports: andrea@cs.st-and.ac.uk

# amount of common subexpressions used:0
# amount of inferred common subexpressions used:0
# amount of original variables saved (over direct equality):0
# amount of constraints:176
# amount of auxiliary variables: 22

**VARIABLES**
DISCRETE V0 {0..11}
DISCRETE V1 {0..11}
DISCRETE V2 {0..11}
DISCRETE V3 {0..11}
DISCRETE V4 {0..11}
DISCRETE V5 {0..11}
DISCRETE V6 {0..11}
DISCRETE V7 {0..11}
DISCRETE V8 {0..11}
DISCRETE V9 {0..11}
DISCRETE V10 {0..11}
DISCRETE V11 {0..11}
DISCRETE V12 {0..10}
DISCRETE V13 {0..10}
DISCRETE V14 {0..10}
DISCRETE V15 {0..10}
DISCRETE V16 {0..10}
DISCRETE V17 {0..10}
DISCRETE V18 {0..10}
DISCRETE V19 {0..10}
DISCRETE V20 {0..10}
DISCRETE V21 {0..10}
DISCRETE V22 {0..10}

# auxiliary variables
DISCRETE aux0 {-11..11}
DISCRETE aux1 {0..11}
DISCRETE aux2 {-11..11}
DISCRETE aux3 {0..11}
DISCRETE aux4 {-11..11}
DISCRETE aux5 {0..11}
DISCRETE aux6 {-11..11}
DISCRETE aux7 {0..11}
DISCRETE aux8 {-11..11}
DISCRETE aux9 {0..11}
DISCRETE aux10 {-11..11}
DISCRETE aux11 {0..11}
DISCRETE aux12 {-11..11}
DISCRETE aux13 {0..11}
DISCRETE aux14 {-11..11}
DISCRETE aux15 {0..11}
DISCRETE aux16 {-11..11}
DISCRETE aux17 {0..11}
DISCRETE aux18 {-11..11}
DISCRETE aux19 {0..11}
DISCRETE aux20 {-11..11}
DISCRETE aux21 {0..11}

# aliases

**SEARCH**

PRINT [
[V0],[V1],[V2],[V3],[V4],[V5],[V6],[V7],
[V8],[V9],[V10],[V11],[V12],[V13],[V14],[V15],
[V16],[V17],[V18],[V19],[V20],[V21],[V22]]

VARORDER [
V0,V1,V2,V3,V4,V5,V6,V7,
V8,V9,V10,V11,V12,V13,V14,V15,
V16,V17,V18,V19,V20,V21,V22,
aux0,aux1,aux2,aux3,aux4,aux5,aux6,aux7,
aux8,aux9,aux10,aux11,aux12,aux13,aux14,aux15,
aux16,aux17,aux18,aux19,aux20,aux21]


**CONSTRAINTS**

weightedsumgeq([1,-1], [V10,V11], aux20)
weightedsumleq([1,-1], [V10,V11], aux20)
abs(aux21,aux20)
sumleq([1,V22], aux21)
sumgeq([1,V22], aux21)
weightedsumgeq([1,-1], [V9,V10], aux18)
weightedsumleq([1,-1], [V9,V10], aux18)
abs(aux19,aux18)
sumleq([1,V21], aux19)
sumgeq([1,V21], aux19)
weightedsumgeq([1,-1], [V8,V9], aux16)
weightedsumleq([1,-1], [V8,V9], aux16)
abs(aux17,aux16)
sumleq([1,V20], aux17)
sumgeq([1,V20], aux17)
weightedsumgeq([1,-1], [V7,V8], aux14)
weightedsumleq([1,-1], [V7,V8], aux14)
abs(aux15,aux14)
sumleq([1,V19], aux15)
sumgeq([1,V19], aux15)
weightedsumgeq([1,-1], [V6,V7], aux12)
weightedsumleq([1,-1], [V6,V7], aux12)
abs(aux13,aux12)
sumleq([1,V18], aux13)
sumgeq([1,V18], aux13)
weightedsumgeq([1,-1], [V5,V6], aux10)
weightedsumleq([1,-1], [V5,V6], aux10)
abs(aux11,aux10)
sumleq([1,V17], aux11)
sumgeq([1,V17], aux11)
weightedsumgeq([1,-1], [V4,V5], aux8)
weightedsumleq([1,-1], [V4,V5], aux8)
abs(aux9,aux8)
sumleq([1,V16], aux9)
sumgeq([1,V16], aux9)
weightedsumgeq([1,-1], [V3,V4], aux6)
weightedsumleq([1,-1], [V3,V4], aux6)
abs(aux7,aux6)
sumleq([1,V15], aux7)
sumgeq([1,V15], aux7)
weightedsumgeq([1,-1], [V2,V3], aux4)
weightedsumleq([1,-1], [V2,V3], aux4)
abs(aux5,aux4)
sumleq([1,V14], aux5)
sumgeq([1,V14], aux5)
weightedsumgeq([1,-1], [V1,V2], aux2)
weightedsumleq([1,-1], [V1,V2], aux2)
abs(aux3,aux2)
sumleq([1,V13], aux3)
sumgeq([1,V13], aux3)
weightedsumgeq([1,-1], [V0,V1], aux0)
weightedsumleq([1,-1], [V0,V1], aux0)
abs(aux1,aux0)
sumleq([1,V12], aux1)
sumgeq([1,V12], aux1)
diseq(V21, V22)
diseq(V20, V22)
diseq(V20, V21)
diseq(V19, V22)
diseq(V19, V21)
diseq(V19, V20)
diseq(V18, V22)
diseq(V18, V21)
diseq(V18, V20)
diseq(V18, V19)
diseq(V17, V22)
diseq(V17, V21)
diseq(V17, V20)
diseq(V17, V19)
diseq(V17, V18)
diseq(V16, V22)
diseq(V16, V21)
diseq(V16, V20)
diseq(V16, V19)
diseq(V16, V18)
diseq(V16, V17)
diseq(V15, V22)
diseq(V15, V21)
diseq(V15, V20)
diseq(V15, V19)
diseq(V15, V18)
diseq(V15, V17)
diseq(V15, V16)
diseq(V14, V22)
diseq(V14, V21)
diseq(V14, V20)
diseq(V14, V19)
diseq(V14, V18)
diseq(V14, V17)
diseq(V14, V16)
diseq(V14, V15)
diseq(V13, V22)
diseq(V13, V21)
diseq(V13, V20)
diseq(V13, V19)
diseq(V13, V18)
diseq(V13, V17)
diseq(V13, V16)
diseq(V13, V15)
diseq(V13, V14)
diseq(V12, V22)
diseq(V12, V21)
diseq(V12, V20)
diseq(V12, V19)
diseq(V12, V18)
diseq(V12, V17)
diseq(V12, V16)
diseq(V12, V15)
diseq(V12, V14)
diseq(V12, V13)
diseq(V10, V11)
diseq(V9, V11)
diseq(V9, V10)
diseq(V8, V11)
diseq(V8, V10)
diseq(V8, V9)
diseq(V7, V11)
diseq(V7, V10)
diseq(V7, V9)
diseq(V7, V8)
diseq(V6, V11)
diseq(V6, V10)
diseq(V6, V9)
diseq(V6, V8)
diseq(V6, V7)
diseq(V5, V11)
diseq(V5, V10)
diseq(V5, V9)
diseq(V5, V8)
diseq(V5, V7)
diseq(V5, V6)
diseq(V4, V11)
diseq(V4, V10)
diseq(V4, V9)
diseq(V4, V8)
diseq(V4, V7)
diseq(V4, V6)
diseq(V4, V5)
diseq(V3, V11)
diseq(V3, V10)
diseq(V3, V9)
diseq(V3, V8)
diseq(V3, V7)
diseq(V3, V6)
diseq(V3, V5)
diseq(V3, V4)
diseq(V2, V11)
diseq(V2, V10)
diseq(V2, V9)
diseq(V2, V8)
diseq(V2, V7)
diseq(V2, V6)
diseq(V2, V5)
diseq(V2, V4)
diseq(V2, V3)
diseq(V1, V11)
diseq(V1, V10)
diseq(V1, V9)
diseq(V1, V8)
diseq(V1, V7)
diseq(V1, V6)
diseq(V1, V5)
diseq(V1, V4)
diseq(V1, V3)
diseq(V1, V2)
diseq(V0, V11)
diseq(V0, V10)
diseq(V0, V9)
diseq(V0, V8)
diseq(V0, V7)
diseq(V0, V6)
diseq(V0, V5)
diseq(V0, V4)
diseq(V0, V3)
diseq(V0, V2)
diseq(V0, V1)

**EOF**
